cmd/oras/pull.go

```diff
@@ -115,12 +116,20 @@ func runPull(opts pullOptions) error {
 					s.Annotations = make(map[string]string)
 				}
 				s.Annotations[ocispec.AnnotationTitle] = configPath
-			} else if s.Annotations[ocispec.AnnotationTitle] == "" {
+			}
+			if s.Annotations[ocispec.AnnotationTitle] == "" {
 				ss, err := content.Successors(ctx, fetcher, s)
 				if err != nil {
 					return nil, err
 				}
 				if len(ss) == 0 {
+					if !printed[s.Digest.String()] {
+						err = display.StatusPrinter("Skipped    ", opts.Verbose)(ctx, s)
+						if err != nil {
+							return nil, err
+						}
+						printed[s.Digest.String()] = true
```

Comment:
What if a content is named in Manifest `A` but unnamed in Manifest `B`?

Reply:
Discussed in the issue.

Reply:
Let's wait for confirm in the issue. Other parts look good to me.